static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.spec.ts

```diff
@@ -1,22 +1,17 @@
+import type { SizeMapping } from '@guardian/commercial-core';
 import { adSizes } from '@guardian/commercial-core';
 import config from '../../../../lib/config';
 import { Advert } from '../dfp/Advert';
-
-import { getHeaderBiddingAdSlots, _ } from './slot-config';
-import {
-	getBreakpointKey as getBreakpointKey_,
-	shouldIncludeMobileSticky as shouldIncludeMobileSticky_,
-} from './utils';
+import { _, getHeaderBiddingAdSlots } from './slot-config';
+import { getBreakpointKey, shouldIncludeMobileSticky } from './utils';
 
 jest.mock('lib/raven');
 
 const { getSlots } = _;
 
-const getBreakpointKey = getBreakpointKey_;
-const shouldIncludeMobileSticky = shouldIncludeMobileSticky_;
-
 jest.mock('./utils', () => {
-	const original = jest.requireActual('./utils');
+	// eslint-disable-next-line -- Eslint doesn't understand jest.requireActual
```

Comment:
🤷 This is the pattern we use to do this in a few other places, not sure there's a way round it...

Reply:
There's this way too, not sure it's much better tbh https://github.com/guardian/frontend/blob/4903ad8763fb2daf9b5cf23b6aa20e2d1aeee2db/static/src/javascripts/projects/commercial/modules/manage-ad-free-cookie-on-consent-change.spec.ts#L22

Reply:
Thanks Jake, let me try that quickly

Reply:
I actually think your solution nicer, as it manages to avoid the lint disabling comment. Thanks!